float8_experimental/float8_linear.py

```diff
@@ -326,11 +390,29 @@ def forward(self, x):
         if self.bias is not None:
             y = y + self.bias.to(y.dtype)
 
-        self.float8_post_forward()
+        if self.has_any_delayed_scaling:
+            self.float8_post_forward()
         return y
 
+    def extra_repr(self):
```

Comment:
should we override the default repr to print this extra repr?

Reply:
ohh nvm from the tests it looks this is already done?

Reply:
yes this is defined on `torch.nn.Module` for exactly this use case